client/channel/channel.go

```diff
@@ -36,6 +36,22 @@ const (
 	prometheusSubsystem = "channel"
 )
 
+// Request represents an request from server.
+// It is similar to agentpb.ServerMessage except it can contain only requests,
+// and the payload is already unwrapped (XXX instead of ServerMessage_XXX).
+type Request struct {
```

Comment:
maybe it's better to rename these two types to `ServerRequest` and `AgentResponse`, because agent can send request to pmm-managed as well.